static/sass/styles.scss

```diff
@@ -227,3 +227,11 @@ $color-social-icon-foreground: $color-light;
 
 @import 'snapcraft_tour';
 @include snapcraft-tour;
+
+.lazyload {
```

Comment:
What is this one for?

Reply:
see https://pypi.org/project/canonicalwebteam.image-template/ 
"Add lazy sizes" header

Reply:
Put this link in a comment of the file please.

Reply:
After doing some QA I think this is actually wrong and it makes it look bad.
What it does it tells the browser to remove any sizing from the images before they load. So even if the img tag is properly sized in HTML (what we do in many places to reduce layout changes) it will be forced to have size of 0 before loading. It makes the page jump when images are loaded which is bad from performance (unnecessary rerenders in browser) and is very badly visible (because images are loaded when user scrolls to them). So perceived performance is actually worse.

Current snapcraft.io (on slow connection) - nothing moves around, images just load:

![current-slow](https://user-images.githubusercontent.com/83575/66207396-f3c74200-e6b2-11e9-86f2-02486b01b0e2.gif)

Lazy loading as currently implemented:
![lazy-slow](https://user-images.githubusercontent.com/83575/66207407-fd50aa00-e6b2-11e9-80e1-6605efb48b7c.gif)

It seems that removing this .lazyload style helps (or we could just reduce it to only do visibility:hidden):
![lazy-slow-width](https://user-images.githubusercontent.com/83575/66207442-15282e00-e6b3-11e9-9c9f-df9cdcd2ccee.gif)

Reply:
The problem with only doing visibility is that when JS is disabled, you get a default image that has a width and height that pushed the non-js icon... I'm happy on snapcraft to do this though - most people have JS right!? And if that's the case, we should just remove this CSS altogether